tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -807,6 +807,12 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiSharedReservationSize:
     conditions:
       - "asic_type not in ['cisco-8000']"
 
+qos/test_qos_sai.py::TestQosSai::testQosSaiLosslessVoq:
+  skip:
+    reason: "Lossless Voq test is not supported in Cisco-8000."
+    conditions:
+      - "asic_type in ['cisco-8000']"
+
```

Comment:
Is this getting skipped for Cisco or non Cisco devices ? As per test_qos_sai.py file, it should be skipped for non Cisco ?

Reply:
My bad. Closed this PR.